frontend/README.md

```diff
@@ -0,0 +1,14 @@
+# URL-S Frontend
+
+Instructions for configuring and starting the URL-S frontend app, Vue.js, and its dependencies.
+
+It is recommended that you install and start the Backend applications before using the Frontend. See the README in the Backend folder.
+
+## Local Setup
+**`Step 1`**. Open a terminal (option to do this in MS Code). Change directory to the frontend/url-shortener folder
```

Comment:
Ask to open a new terminal for frontend opreations

Reply:
Fixed in commit = Update README.md to clarify env file must be created in the backend folder